src/Resources/RetailerDemListResource.cs

```diff
@@ -11,5 +11,7 @@ public class RetailerDemListResource : HypermediaResource
         public string LastReviewedOn { get; set; }
 
         public IEnumerable<RootProductResource> RootProducts { get; set; }
+
+        public IEnumerable<RetailerDemListActivityResource> Activities { get; set; }
```

Comment:
Most of our recent codebases have featured a seperate URI to request activities rather than bundling them on to the parent resource - this allows consuming services that don't care about the history to just not request it, thus improving performance.